k8s/k8s.go

```diff
@@ -117,7 +182,7 @@ func CmdAddK8s(args *skel.CmdArgs, conf utils.NetConf, hostname string, calicoCl
 		endpoint.Metadata.Node = hostname
 		endpoint.Metadata.Orchestrator = orchestrator
 		endpoint.Metadata.Workload = workload
-		endpoint.Metadata.Labels = make(map[string]string)
```

Comment:
This is actually a change in the value that's being assigned to labels here for the non-k8s policy case, and IIRC I think it will break stuff.

Take a look here:

https://play.golang.org/p/Ucwd7q07Xk